pennylane/qchem/tapering.py

```diff
@@ -350,9 +353,8 @@ def taper(h, generators, paulixops, paulix_sector):
 
     >>> symbols = ["H", "H"]
     >>> geometry = np.array([[0.0, 0.0, -0.69440367], [0.0, 0.0, 0.69440367]])
-    >>> mol = qml.hf.Molecule(symbols, geometry)
-    >>> H = qml.hf.generate_hamiltonian(mol)(geometry)
-    >>> generators= qml.hf.symmetry_generators(H)
+    >>> H, q = qml.qchem.molecular_hamiltonian(symbols, geometry)
```

Comment:
```suggestion
    >>> H, qubits = qml.qchem.molecular_hamiltonian(symbols, geometry)
```

Reply:
Nice paying attention to details in code review @obliviateandsurrender